### ISSUE TITLE
Unable to Install with Node 4.1.1 and NPM 3.5.3

### ISSUE THREAD

**Issue by leopoldjoy** [unclecheese/react-selectable#7]
When I try to install `react-selectable` into my ReactJS application using:

```
npm install react-selectable
```

I get this error:

```
myapp@0.1.0 /pathToApp/
├─┬ babel@5.8.35
│ └─┬ chokidar@1.4.2
│   └─┬ fsevents@1.0.7
│     └─┬ node-pre-gyp@0.6.19
│       └─┬ request@2.67.0
│         └── forever-agent@0.6.1 
├── crc@3.4.0 
├── forever-agent@0.6.1 
├── mime@1.3.4 
├── node-sass@3.4.2  extraneous
├── radium@0.16.5 
├── react-textarea-autosize@3.3.0  extraneous
└── sass-loader@3.1.2  extraneous

npm WARN enoent ENOENT: no such file or directory, open '/pathToApp/node_modules/acorn-globals/package.json'
npm WARN enoent ENOENT: no such file or directory, open '/pathToApp/node_modules/escodegen/package.json'
npm WARN enoent ENOENT: no such file or directory, open '/pathToApp/node_modules/spawn-sync/package.json'
npm ERR! Darwin 13.2.0
npm ERR! argv "/usr/local/Cellar/node/5.5.0/bin/node" "/usr/local/bin/npm" "install" "react-selectable"
npm ERR! node v5.5.0
npm ERR! npm  v3.5.3
npm ERR! path /pathToApp/node_modules/react-selectable/node_modules/webpack/bin/webpack
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/pathToApp/node_modules/react-selectable/node_modules/webpack/bin/webpack'
npm ERR! enoent ENOENT: no such file or directory, chmod '/pathToApp/node_modules/react-selectable/node_modules/webpack/bin/webpack'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /pathToApp/npm-debug.log
```

I'm not sure why I'm getting this, any ideas?

**Comment by leopoldjoy** [comment]
Not sure what changed, but it installed fine this morning without issue.

**Comment by unclecheese** [comment]
Really? I've been working on it. Made a few commits, but I'm getting errors now that says there are two `React` libraries installed. Maybe it's unrelated.

**Comment by unclecheese** [comment]
Several fixes in `0.2.5` should get this running more smoothly. Thanks for your patience!

**Comment by leopoldjoy** [comment]
Yeah, `0.2.5` fixed the issue. Thanks again!
